### DISCUSSION TITLE
Difference between python class creation and create

### DISCUSSION THREAD

**Discussion by EngRaff92** [pyuvm/pyuvm#67] [General]
Hello all, speaking about pyuvm is there any difference between the create method call and the python way to create a class ? I know it makes difference in uvm but since there is no factory registration for comp and obj i would guess they are equal am i right? If so what are the differences? Regards

**Comment by raysalemi** [comment]
**pyuvm** registers all UVM classes to the factory without the macros from SystemVerilog.  So `create()` creates the object with the factory as you would expect. 

Here is a blog post with details: https://blogs.sw.siemens.com/verificationhorizons/2021/11/17/uvm-factory/

**Reply by EngRaff92** [reply]
I perfectly get thanks for the answer and the link
